src/components/SelectV2/Select.tsx

```diff
@@ -191,7 +191,7 @@ export const Select = ({
       >
         {options.map(({ startAdornment, ...option }, index) => (
           <option key={index} {...option}>
-            {option.label || option.value}
+            {option?.label || option.value}
```

Comment:
If `option` would be undefined, it would also be true for `option.value`. That said, I don't think option can be undefined here, since it's a rest operator, it would always default to an empty object 🤔